examples/ipfs-kad/src/main.rs

```diff
@@ -66,9 +67,24 @@ async fn main() -> Result<(), Box<dyn Error>> {
         SwarmBuilder::with_async_std_executor(transport, behaviour, local_peer_id).build()
     };
 
+    let cli_opt = Opt::parse();
+
+    match cli_opt {
+        Opt {
+            argument: CliArgument::GetPeers { peer_id },
+        } => get_closes_peers(&mut swarm, peer_id).await,
+        Opt {
+            argument: CliArgument::PutPkRecord {},
+        } => insert_pk_record(&mut swarm, local_peer_id, local_key).await,
+    }
+}
+
+async fn get_closes_peers<TStore: kad::store::RecordStore + Send + 'static>(
+    swarm: &mut Swarm<kad::Behaviour<TStore>>,
```

Comment:
```suggestion
    swarm: &mut Swarm<kad::Behaviour<kad::store::MemoryStore>>,
```

Happy to not be generic here.